fvm/environment/contract_updater_test.go

```diff
@@ -78,8 +80,9 @@ func TestContract_ChildMergeFunctionality(t *testing.T) {
 
 	// rollback
 	err = contractUpdater.SetContract(
-		address,
-		"testContract2",
+		common.AddressLocation{
+			Name:    "testContract2",
```

Comment:
Can you explain what is the name field of a `AddressLocation` for? Why an address location is an address + a name?

Reply:
I see it as a named location on an address. This type is defined and used by cadence mostly for contracts i think. It might be worth checking with cadence if it would make sense to rename it to contract location if that is the only thing it is being used for.